pkg/controllers/user/monitoring/clusterHandler.go

```diff
@@ -365,6 +365,9 @@ func (ch *clusterHandler) deployApp(appName, appTargetNamespace string, appProje
 		return nil, err
 	}
 
+	// clean overwrote answers from annotation
+	monitoring.DropOverwroteAppAnswers(cluster.Annotations)
```

Comment:
I don't understand why you are doing this. Please explain a little.

Reply:
After applying the overwrote answers:
1. Modify the monitoring apps from Catalog Apps page
2. Something triggers the monitoring sync
3. The stale answers will overwrite the new modification from 1

So I think cleaning the overwrote answers to keep the modification from Catalog Apps page.

Reply:
But Should we update cluster object for it? It is like you are modifying the cache but not updating the object.

Reply:
NVM, I see it now